Makeconf

```diff
@@ -4,6 +4,15 @@
 ## 1. Edit Section 1 and 2 below
 ## 2. Execute: gmake install
 
+## To install ESS for yourself:
+## 1. cat > Makeconf.local -<<EOF
+##    PREFIX=/home/<user>/.local
+##    SITELISP=/home/<user>/.emacs.d/elpa
+##    INFODIR=$(LISPDIR)
+##    ETCDIR=$(LISPDIR)/etc
+##    EOF
+## 2. Execute: gmake install
```

Comment:
Why not `source Makeconf.local` before the `gmake install` without us bothering with it?

Reply:
Sure, that's possible but taking this to its logical conclusion I could also put them in my `.bashrc`.
As an old timer (and which emacs and R users aren't?), I too am accustomed to the traditional download-untar-{xmkmf,configure,autogen,etc} way of installation.  But we're now in an age of "configuration as source" and having `git diff` show my customizations to `Makeconf` is imo unsettling.  `Makeconf.local` seemed like the right balance of tastefulness and not going whole hog with a gnu autotools solution (Makefile.am, configure.in, etc.).  I'm perfectly okay with its rejection, however.

Reply:
> git diff show my customizations to Makeconf is imo unsettling. 

I completely agree. Makeconf setup as we have doesn't seem make much sense these days. It's much more fruitful to pick local configuration of some kind.

@mmaechler, @jabranham, @lionel-  would you be against  removing Makeconf in favor of dir local and user root local .ess-makeconfig (or .ess-makevars)?